packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx

```diff
@@ -144,6 +173,15 @@ export class FbFieldDropdownItem {
       : int + ordinals[3];
   };
 
+  private nameFocusHandler = (event: CustomEvent) => {
+    event.stopImmediatePropagation();
```

Comment:
Can you check the condition if(!this.isDependentField) { return; } here as we dont need this for other dropdown fields

Reply:
Done.